fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -36,12 +36,23 @@ class ConditionalSelectionRate(ClassificationMoment):
     where error(A = a) = total_error
     """
 
-    def __init__(self, ratio=1.0):
+    def __init__(self, difference_bound=None, ratio_bound_slack=None, ratio_bound=None):
```

Comment:
Checked with @MiroDudik --> default `None` is fine, except `ratio_bound` should be the second arg and `ratio_bound_slack` should be `0.0` by default.

Then inside the method as discussed we should think about backwards compatibility. What that means for us is that in the case where both `difference_bound` and `ratio_bound` are None we use a default constant (should be defined at the top of the file)
```python
_DEFAULT_DIFFERENCE_BOUND = 0.01
```
which will replicate the behavior we have so far. We wouldn't want users to suddenly be surprised with failures for `DemographicParity()` etc.. So that kind of replaces the action after the check for whether both are None.
The check for whether both are not None is still valid, of course!

Reply:
That makes sense. Just to clarify, the slack is a default for epsilon (0.0), but the ratio bound r should always be provided (with the exception of the backwards compatibility)? 

From the proposal: 
ratio_bound=r, ratio_bound_slack=ε

Reply:
Defined global variable _DEFAULT_DIFFERENCE_BOUND = 0.01 and initialized self.eps = _DEFAULT_DIFFERENCE_BOUND if neither difference_bound nor ratio_bound are provided. Please let me know if that's an incorrect interpretation

Reply:
@afehmiu that's correct!

@laurenpendo When using the difference bound then epsilon is that bound.

Reply:
@romanlutz sounds good-- I already did that just wanted to make sure my logic was correct, thanks!